app/services/qa/linked_data/request_header_service.rb

```diff
@@ -26,6 +27,7 @@ def search_header
         header[:user_language] = user_language
         header[:performance_data] = performance_data?
         header[:context] = context?
+        header[:response_header] = response_header?
```

Comment:
Adds parameter to QA request URL for user to specify they want the results to include a response_header with pagination info.